hip/base/hipsparse_bindings.hip.hpp

```diff
@@ -457,6 +588,23 @@ inline void destroy(hipsparseMatDescr_t descr)
 }
 
 
+inline csrgemm2Info_t create_spgemm_info()
+{
+    csrgemm2Info_t info{};
+    GKO_ASSERT_NO_HIPSPARSE_ERRORS(hipsparseCreateCsrgemm2Info(&info));
+    return info;
+}
+
+// on HIP, hipsparseMatDescr_t and csrgemm2Info_t are both void*, so this would
+// be a redefinition
+#ifndef __HIP_PLATFORM_HCC__
+inline void destroy(csrgemm2Info_t info)
```

Comment:
If the platform is HCC now, will `destroy(csrgemm2Info_t)` call `destroy(hipsparseMatDescr_t)`?

Reply:
That is actually a good point, I assumed that both function didn't do anything, but it might make sense to use different function names here.